examples/workers/kv-todos/model.ts

```diff
@@ -1,11 +1,12 @@
 import * as DB from 'worktop/kv';
 import { uid as toUID } from 'worktop/utils';
+import type { UID } from 'worktop/utils';
 import type { KV } from 'worktop/kv';
 
 declare const TODOS: KV.Namespace;
 
 export interface Todo {
-	uid: string;
+	uid: UID<8>;
```

Comment:
If this had been `UID<11>` (or anything other than `UID<8>` then there'd be type errors throughout this file.